src/entity/task/vdaf.rs

```diff
@@ -0,0 +1,67 @@
+use crate::json_newtype;
+use serde::{Deserialize, Serialize};
+use std::collections::HashSet;
+use validator::{Validate, ValidationError, ValidationErrors};
+
+#[derive(Serialize, Deserialize, Validate, Debug, Clone, Eq, PartialEq)]
+pub struct Histogram {
+    #[validate(required, custom = "sorted", custom = "unique")]
+    pub buckets: Option<Vec<i32>>,
+}
+
+fn sorted(buckets: &Vec<i32>) -> Result<(), ValidationError> {
+    let mut buckets_cloned = buckets.clone();
+    buckets_cloned.sort_unstable();
+    if &buckets_cloned == buckets {
+        Ok(())
+    } else {
+        Err(ValidationError::new("sorted"))
+    }
+}
+
+fn unique(buckets: &Vec<i32>) -> Result<(), ValidationError> {
```

Comment:
nit: if you take the above suggestion, we could also fuse these two validators into something like `sorted_and_unique`: for each consecutive pair of elements `a`, `b`, validate that `a < b`. in addition to the benefits to `sorted` above, this would also drop the `HashSet` allocation from `unique`.

Reply:
I think we wouldn't get different validation messages then

Reply:
Regardless, this code isn't new in this PR, it just moved into its own vdaf.rs because task.rs was getting too big